### ISSUE TITLE
post_install_script_uri lead to instance crash while running ibm_compute_vm_instance

### ISSUE THREAD

**Issue by Otterton1978** [IBM-Cloud/terraform-provider-ibm#249]
Hi there,
Here I report an issue I found when running ibm_compute_vm_instance to deploy VM in IBM Cloud. Kindly look into my report to get it resolved. Thanks.

### Terraform Version
Run `terraform -v` to show the version. If you are not running the latest version of Terraform, please upgrade because your issue may have already been fixed.

### Affected Resource(s)
Please list the resources as a list, for example:
- ibm_compute_vm_instance

If this issue appears to affect multiple resources, it may be an issue with Terraform's core, so please mention this.

### Terraform Configuration Files
```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.

provider "ibm" {
    softlayer_username = "${var.ibm_sl_username}"
    softlayer_api_key  = "${var.ibm_sl_api_key}"
}

variable "count" {
  default = 1
}

resource "ibm_compute_vm_instance" "tf-vm" {
    count = "${var.count}"
    hostname = "${format("tf-test-%03d", count.index + 1)}"
    domain = "sk.com"
    datacenter = "seo01"
    os_reference_code = "WIN_2012-STD-R2_64"
    flavor_key_name = "B1_2X4X100"
    local_disk = false
    network_speed = 100
    hourly_billing = true
    post_install_script_uri = "https://raw.githubusercontent.com/mungi/gist/master/test.ps1"
    tags = ["poc","test","vm","windows","${format("Server%03d", count.index + 1)}"]
#    wait_time_minutes = 10
    notes = "My description for this VM"
}

```

### Debug Output
Please provider a link to a GitHub Gist containing the complete debug output: https://www.terraform.io/docs/internals/debugging.html. Please do NOT paste the debug output in the issue; just paste a link to the Gist.

https://gist.github.com/Otterton1978/f0589978307190ac488edfb66e41b25e

### Panic Output
If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`.

https://gist.github.com/Otterton1978/11ee88ec431cc07a8581b1d8eb6a7fc4

### Expected Behavior
What should have happened?

Run post provision script without error

### Actual Behavior
What actually happened?

Terraform crashed when post_install_script_uri is specified in the config.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. Resource "ibm_compute_vm_instace" to be used to order new Windows hourly VSI
2. include "post_install_script_uri" in the resource to run post provision script
3. terraform apply

### Important Factoids
Are there anything atypical about your accounts that we should know? For example: Running in EC2 Classic? Custom version of OpenStack? Tight ACLs?

Using Terraform and Ansible to automate provision and application setup process in IBM Cloud

### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:
- GH-1234

**Comment by sakshiag** [comment]
@Otterton1978 : The issue is with WINDOWS VM only while reading the disks value when provisioned with post_install_script_uri.. I have noticed it is having the third disk as CD-ROM which has no capacity defined which results into the error. I have posted the query in softlayer. Once we will get the reply will update. 
Thanks

**Comment by sakshiag** [comment]
@Otterton1978 : Fix has been merged in the master's.  You can take the latest code and build the plugin.

**Comment by sakshiag** [comment]
Available as part of v0.10.0 release https://github.com/IBM-Cloud/terraform-provider-ibm/releases/tag/v0.10.0
